website/docs/tailwind/helpers.md

```diff
@@ -0,0 +1,150 @@
+---
+id: helpers
+title: Helpers
+---
+
+Frontend libs Tailwind includes many useful helpers.
+
+Here are a few featured:
+
+## General helpers
+
+### `getHiddenOptions`
+In order to simplify hiding parts of component options, the `getHiddenOptions` helper was created.
+
+To use, first in your component add:
+```jsx
+export const DemoOptions = (attributes) => {
+	const {
+		setAttributes,
+		// highlight-next-line
```

Comment:
Curious to se how this looks in action... can you please provide a live example

Reply:
Playground 😉

Reply:
Found it, not in the playground tho' 😄 
![image](https://github.com/infinum/eightshift-docs/assets/22823970/b59f1b08-6790-44b2-af5e-ef211e38c89e)

Reply:
Not sure what you have in mind with the "interactive"

Reply:
Ah, I thought you meant you want an interactive example hahah